cmd/snap/cmd_services.go

```diff
@@ -138,15 +138,15 @@ func (s *svcStatus) Execute(args []string) error {
 	fmt.Fprintln(w, i18n.G("Service\tStartup\tCurrent\tNotes"))
 
 	for _, svc := range services {
-		startup := i18n.G("disabled")
```

Comment:
Hmm not sure, I would say that scripts should set LC_ALL=C or one of the other variables.

Reply:
I made this as a suggestion in the bug instead, @stolowski I'm closing this instead.